addons/web/static/src/js/views/form/form_controller.js

```diff
@@ -519,8 +521,10 @@ var FormController = BasicController.extend({
             var unarchive_item = _.find(this.sidebar.items.other, function(item) {
                 return item.classname && item.classname.includes('o_sidebar_item_unarchive')
             })
+            const activeField = this.model._getActiveField(this.renderer.state);
```

Comment:
Typically, we don't access the state of the renderer, so I'd suggest:
```js
const state = this.model.get(this.handle, {raw: true});
const activeField = this.model._getActiveField(state);
```
Or you could still use this.initialState as your are interested in an invariable property.